### ISSUE TITLE
Need Understanding WCAG 2.1 and esp SUFFICIENT techniques to be developed at same time as SC

### ISSUE THREAD

**Issue by GreggVan** [w3c/wcag21#279]
With WCAG 2.0, the Understanding WCAG 2.0  and the techniques were developed in parallel and released at the same time. 

SC were not added to a draft we released until these sufficient techniques for meeting them were already documented. 

That allowed reviewers to understand what the working group had in mind as being sufficient. 

It also allow the working group to discover when things that sounded good were not practical because the working group could not figure out techniques to address the different cases that it, or reviewers, raised. 

By developing techniques for different technologies it also allowed the working group to figure out when something did not work across technologies. 

 I would suggest the same practice fused with WCAG 2.1.   SC should not be added to the draft unless the understanding document and especially the sufficient techniques for meeting it are documented first.  We found that it guaranteed to be instructive each time we did it (or in those cases when we tried it and we failed to find techniques and had to rethink our SC).

**Comment by michael-n-cooper** [comment]
I'm working on getting Understanding and Techniques set up in time to include with the next TR publication (fingers crossed), for the reasons you explain. There is draft content for Understanding, and indications of potential techniques for most of the SC proposals, because we requested it as part of the template, so it's mainly a matter of getting it into a location we can develop and expose for public review more.

**Comment by awkawk** [comment]
Thanks for the advice, and much more of the work will be taking place in parallel moving forward.
